platform/ios/Integration Tests/MGLMapViewIntegrationTest.m

```diff
@@ -1,5 +1,9 @@
 #import "MGLMapViewIntegrationTest.h"
 
+@interface MGLMapView (MGLMapViewIntegrationTest)
```

Comment:
There is no compile time enforcement that a method declared in a named category will actually exist at run time. This is a bit of a code organization challenge with Objective-C

This is test code, and the tests would fail with a crash if this method were renamed or removed, so IMO it's not a show-stopper. Just something to be aware of. I believe we can work around this issue with a different approach to organizing the declaration and definition of this method but I'm not entirely convinced it's needed.

Reply:
It would be nice if there was an Obj-C equivalent of Swift's `@testable`. We could add the method to `MGLMapView_Private.h` but then this causes more issues. I think we can leave it as it is for the moment, aware of the deficiencies .